salvo/src/lib/source_tree.py

```diff
@@ -0,0 +1,207 @@
+"""This module contains methods for managing a source tree"""
+
+import logging
+import os
+import re
+from subprocess import CalledProcessError
```

Comment:
Can we import `subprocess` instead to follow the guidance of importing packages and modules only?

https://github.com/envoyproxy/envoy-perf/blob/9fd6a9b18943750bb674b2543210559e5c9fe58c/salvo/src/lib/docker/test_docker_image.py#L11-L12

Note - import from the typing module are specifically listed as an exception to this rule.

Reply:
Sure